client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -251,14 +251,18 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
             if (result && result.format) {
                 const data = this.isMultiSelect ? this.selectedRows : this.dataSource.filteredData;
                 if (result.format === 'pdf') {
-                    this.pdfExport.exportMotionCatalog(
-                        data,
-                        result.lnMode,
-                        result.crMode,
-                        result.content,
-                        result.metaInfo,
-                        result.comments
-                    );
+                    try {
+                        this.pdfExport.exportMotionCatalog(
+                            data,
+                            result.lnMode,
+                            result.crMode,
+                            result.content,
+                            result.metaInfo,
+                            result.comments
+                        );
+                    } catch (err) {
+                        this.raiseError(err.message);
```

Comment:
`if (err instanceof PdfError) {`. Otherwise you catch every exception, so your custom class won't make any sense, because you could just throw an `Error`.